test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -1024,3 +1024,32 @@ func.func @torch.aten.baddbmm(%arg0: !torch.vtensor<[?,?,?],f32>, %arg1: !torch.
   %0 = torch.aten.baddbmm %arg0, %arg1, %arg2, %int1, %int1 : !torch.vtensor<[?,?,?],f32>, !torch.vtensor<[?,?,?],f32>, !torch.vtensor<[?,?,?],f32>, !torch.int , !torch.int -> !torch.vtensor<[?,?,?],f32>
   return %0 : !torch.vtensor<[?,?,?],f32>
 }
+
+// CHECK-LABEL:   func.func @torch.aten.repeat(
+// CHECK-SAME:                 %[[ARG0:.*]] !torch.vtensor<[3,1],f32>) -> !torch.vtensor<[2,12,5],f32> {
+// CHECK            %[[INT2:.*]] = torch.constant.int 2
+// CHECK            %[[INT4:.*]] = torch.constant.int 4
+// CHECK            %[[INT5:.*]] = torch.constant.int 5
+// CHECK            %[[T0:.*]] = torch.prim.ListConstruct %int2, %int4, %int5 : (!torch.int, !torch.int, !torch.int) -> !torch.list<int>
```

Comment:
Actually, I am talking about captured variables. In this case, you have captured `torch.constant.int 2` in a variable `INT2` but you haven't passed that variable in `torch.prim.ListConstruct`. It's still `%int2` over there instead of `[[INT2]]`.